middle_end/flambda2/to_cmm/to_cmm_expr.ml

```diff
@@ -668,7 +675,8 @@ and let_cont_rec env res invariant_params conts body =
       conts_to_handlers
       (Continuation.Map.empty, res)
   in
-  let dbg = Debuginfo.none (* CR mshinwell: fix debuginfo *) in
+  (* CR mshinwell: fix debuginfo *)
+  let dbg = Env.add_inlined_debuginfo env Debuginfo.none in
```

Comment:
Same as above (if we do change this, maybe leave CRs noting that we need to call add_inlined_debuginfo when we do end up with the correct dbg one day)

Reply:
fixed